src/components/PeopleFilter/PeopleFilter.tsx

```diff
@@ -0,0 +1,138 @@
+import { useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
+import React from 'react';
+import { SearchLink } from '../SearchLink';
+
+export const PeopleFilters = React.memo(() => {
+  const centuriesValues = ['16', '17', '18', '19', '20'];
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+  const sex = searchParams.get('sex');
+
+  const updateSearchParams = (
+    params: { [key: string]: string | string[] | null },
+  ) => {
+    Object.entries(params).forEach(([key, value]) => {
+      if (!value) {
+        searchParams.delete(key);
+      } else if (Array.isArray(value)) {
+        searchParams.delete(key);
+        value.forEach(part => {
+          searchParams.append(key, part);
+        });
+      } else {
+        searchParams.set(key, value);
+      }
+    });
+
+    setSearchParams(searchParams);
+  };
```

Comment:
We have the same function in utils file searchHelper.ts, why did you create your own?

Reply:
i just wanted to do it myself ))) u a right its the same functionality, but I did it in purpose